migration/migrator/data/submitty_db.sql

```diff
@@ -416,6 +435,7 @@ CREATE TABLE public.courses_users (
     registration_section character varying(255),
     registration_type character varying(255) DEFAULT 'graded'::character varying,
     manual_registration boolean DEFAULT false,
+    previous_registration_section character varying,
```

Comment:
Should this be character varying(255) as well if the source where it would come from is also 255?

Reply:
Is 255 the character limit? Gotcha!